specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/TextAnalytics.json

```diff
@@ -0,0 +1,3055 @@
+          "NLCitizensServiceNumberV2",
+          "NLTaxIdentificationNumber",
+          "NLValueAddedTaxNumber",
+          "NZBankAccountNumber",
+          "NZDriversLicenseNumber",
+          "NZInlandRevenueNumber",
+          "NZMinistryOfHealthNumber",
+          "NZSocialWelfareNumber",
+          "NOIdentityNumber",
+          "PHUnifiedMultiPurposeIDNumber",
+          "PLIdentityCard",
+          "PLNationalID",
+          "PLNationalIDV2",
+          "PLPassportNumber",
+          "PLTaxIdentificationNumber",
+          "PLREGONNumber",
+          "PTCitizenCardNumber",
+          "PTCitizenCardNumberV2",
+          "PTTaxIdentificationNumber",
+          "ROPersonalNumericalCode",
+          "RUPassportNumberDomestic",
+          "RUPassportNumberInternational",
+          "SANationalID",
+          "SGNationalRegistrationIdentityCardNumber",
+          "SKPersonalNumber",
+          "SITaxIdentificationNumber",
+          "SIUniqueMasterCitizenNumber",
+          "ZAIdentificationNumber",
+          "KRResidentRegistrationNumber",
+          "ESDNI",
+          "ESSocialSecurityNumber",
+          "ESTaxIdentificationNumber",
+          "SQLServerConnectionString",
+          "SENationalID",
+          "SENationalIDV2",
+          "SEPassportNumber",
+          "SETaxIdentificationNumber",
+          "SWIFTCode",
+          "CHSocialSecurityNumber",
+          "TWNationalID",
+          "TWPassportNumber",
+          "TWResidentCertificate",
+          "THPopulationIdentificationCode",
+          "TRNationalIdentificationNumber",
+          "UKDriversLicenseNumber",
+          "UKElectoralRollNumber",
+          "UKNationalHealthNumber",
+          "UKNationalInsuranceNumber",
+          "UKUniqueTaxpayerNumber",
+          "USUKPassportNumber",
+          "USBankAccountNumber",
+          "USDriversLicenseNumber",
+          "USIndividualTaxpayerIdentification",
+          "USSocialSecurityNumber",
+          "UAPassportNumberDomestic",
+          "UAPassportNumberInternational",
+          "Organization",
+          "Email",
+          "URL",
+          "Age",
+          "PhoneNumber",
+          "IPAddress",
+          "Date",
+          "Person",
+          "Address",
+          "All",
+          "Default"
+        ]
+      },
+      "type": "array",
+      "uniqueItems": true
+    },
+    "EntityResolutions": {
+      "properties": {
+        "dateTime.TimeRange": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeRangeResolution"
+          }
+        },
+        "dateTime.Time": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeResolution"
+          }
+        },
+        "quantity.Dimension": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DimensionResolution"
+          }
+        }
+      }
+    },
+    "TimeRangeResolution": {
+      "type": "object",
+      "properties": {
+        "timex": {
```

Comment:
Will it be typical to include multiple representations within a single resolution type?

Reply:
Yes, for example if the query is: "Let's meet at 6'o clock" and it is not clear whether this is a.m or p.m, we include both, hence the array definition

Reply:
I think what I meant by this is that it looks like we have two different of representations here within the same object, one that is expressed as a timex expression and another that is expressed using the `start` and `end` properties.  I was wondering how common of a pattern this will be and if there is some way we can organize it consistently across different types.

Reply:
As a follow up to the last comment, is this something that should be considered as the `value` for `TimeRangeResolution`?  I think this is also asked in another comment I made previously as well.

Reply:
Sorry, I misunderstood your first question. My answer: It is not necessarily multiple representations. It's more like a single representation expressed by multiple properties. For instance, you express a resolution for Speed through "unit" and "value" as in 50 Mph. 
Or more broadly, there might be a resolution for an entity such as Person name with the following set of properties: {first name, middle name, last name}. 

I'm yet to work these details with the PM team. Specifically about the TimeRangeResolution, I am going to put a value property there and fill it with ISO8601 representation which is a widely accepted format and in the meantime, I will let the PM team to decide whether they want to keep the legacy properties such as "timex", "start" and "end" or not. Because, honestly these properties are consumed by all Luis customers and getting rid of them will be breaking to them.

Reply:
Ok, I will resolve for now.  We can discuss in more detail during the API review.  :)

Reply:
@abhahn I actually have an update on this. We are going to produce standard outputs for the date time values. For instance, instead of "start" and "end" we will generate a TimeSpan.